src/questionGenerator.js

```diff
@@ -73,3 +73,56 @@ function displayQuestion(question) {
 const generateButton = document.getElementById('generateButton');
 
 generateButton.addEventListener('click', handleButtonClick);
+
+//brick animation below//
+
+const brickContainer = document.getElementById("brick-container")
+
+const brickContent = [ '&#129521', '&#x1F9F1', ]
+
+const random = (num) => {
+  return Math.floor(Math.random() * num);
+}
+
+const getRandomStyles = () => {
+  const top = random(100);
+  const left = random(100);
+  const dur = random(10) + 10;
+  const size = random(25) + 25;
+  return ` 
+top: -${top}%; 
+left: ${left}%; 
+font-size: ${size}px; 
+animation-duration: ${dur}s; 
+`;
+} 
+
+const createBrick = (num) => {
+  for (var i = num; i > 0; i--) {
+    var brick = document.createElement("div");
```

Comment:
Would replace the `var`s with `let`s here

[Good explainer here](https://codeburst.io/javascript-var-let-or-const-which-one-should-you-use-2fd521b050fa)